### PR TITLE
Apparatus source tooltip popup

### PR DESCRIPTION
**Pull request by ancientTexts-net**
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-box3/greek-apparatus-app/46)

### CODE REVIEW

File: src/components/ApparatusData.js [hunk 1]
```diff
@@ -31,7 +33,18 @@ export default function ApparatusData({ filteredVariantObjects }) {
                     } else {
                       return (
                         <div key={`${i}-source`} className='block ml-2'>
-                          {currentSource.title}
+                          {
+                            currentSource.title.split(' ').map(
+                              (src) => (
+                                <span className="tooltip mr-1">
```
Comment (richmahn) [inline src/components/ApparatusData.js]: Needs a key. Change to:

`                                <span key={`${i}-source-${j}-title`} className="tooltip mr-1">`

**Review by richmahn — CHANGES_REQUESTED** [review]

File: src/components/ApparatusData.js (hunk 1 above)
Comment (richmahn) [reply src/components/ApparatusData.js]: Also, add `, j` to the map params on line 38

**Review by ancientTexts-net — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/unfoldingword-box3/greek-apparatus-app/46)* status: 0 of 3 files reviewed, 1 unresolved discussion (waiting on @richmahn)

___
*[src/components/ApparatusData.js, line 39 at r1](https://reviewable.io/reviews/unfoldingword-box3/greek-apparatus-app/46#-MOPqiafXly3n50byZfm-r1-39:-MOWMEwKC2lpKzOc66sI:b-896fix) ([raw file](https://github.com/unfoldingword-box3/greek-apparatus-app/blob/09f04ebf45409281fd1a44e5e3b280311e131635/src/components/ApparatusData.js#L39)):*
<details><summary><i>Previously, richmahn (Richard Mahn) wrote…</i></summary><blockquote>

Also, add `, j` to the map params on line 38
</blockquote></details>

Done.

**Review by richmahn — APPROVED** [review]
LGTM

**Review by richmahn — APPROVED** [review]
:lgtm:

> Reviewed 2 of 3 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/unfoldingword-box3/greek-apparatus-app/46#-:-MOWQ4gv3e9IKPR7WkQM:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ancientTexts-net)

**Pull request merged**